dotnet/samples/Concepts/TextSearch/TextSearchPluginExample.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Plugins;
+using Microsoft.SemanticKernel.Plugins.Web.Bing;
+
+namespace TextSearch;
+
+/// <summary>
+/// This example shows how to create and use a <see cref="TextSearchPlugin{T}"/>.
+/// </summary>
+public sealed class TextSearchPluginExample(ITestOutputHelper output) : BaseTest(output)
```

Comment:
Could you show how to use it with AzureAISearch - especially how to pass `IndexName`